src/components/mnemonic/mnemonic-input/mnemonic-input.tsx

```diff
@@ -38,9 +39,10 @@ export const MnemonicInput: FC<MnemonicProps> = ({
     <View style={MnemonicStyles.container}>
       <StyledTextInput
         ref={inputRef}
-        keyboardType="visible-password"
         autoCorrect={false}
         autoComplete="off"
+        spellCheck={false}
+        {...(isAndroid ? { keyboardType: 'visible-password' } : {})}
```

Comment:
Previously you did change `[mnemonic-input.tsx](https://github.com/madfish-solutions/templewallet-mobile/pull/390/files#diff-581bc600344cd695f22c41cccfd5c9cfd34a359fa76e6992d73e70b4e69ef8d6)` and [form-text-input.tsx](https://github.com/madfish-solutions/templewallet-mobile/pull/390/files#diff-d7c02609119fe2b1180ddd9e304b49841114b1963b6effebf8338ca078cbd177).  
Don'y you need to do the same with `form-text-input.tsx`?